tests/v2/validation/rbac/rbac.go

```diff
@@ -257,3 +259,78 @@ func createRole(client *rancher.Client, context string, roleName string, rules [
 	return
 
 }
+
+func waitForActiveCluster(client *rancher.Client, clusterID string) error {
```

Comment:
Can we make this an extension? This seems to be in use for almost all of the cluster creation/cluster edits?

Reply:
Moved it to clusters extension